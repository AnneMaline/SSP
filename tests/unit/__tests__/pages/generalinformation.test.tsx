import GeneralInformationPage from "@/app/General-information/page";
import { render } from "@testing-library/react";

describe("General information Page", () => {
  it("renders correctly", () => {
    const { container } = render(<GeneralInformationPage />);
    expect(container).toMatchSnapshot();
  });
});
