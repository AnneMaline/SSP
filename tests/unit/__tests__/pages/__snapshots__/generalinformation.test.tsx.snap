// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`General information Page renders correctly 1`] = `
<div>
  <div>
    <h1>
      General Information
    </h1>
  </div>
</div>
`;
